packages/expo/src/web/Image/AssetSourceResolver.ts

```diff
@@ -0,0 +1,117 @@
+import { Platform } from 'expo-core';
```

Comment:
Put .web.ts extensions on all these files so it's clearer that they should not be imported outside of web

Reply:
TS seems to have a problem with this. These files are agnostic shims which aren't ever referenced directly. Ideally we would move this `web/` folder into another library

Reply:
My understanding is that this file is "swapped in" with the webpack alias rule -> I don't quite understand what this does to TS, could you explain a bit more?

Reply:
If there is no default file ex: `foo.ts` then I cannot `import 'foo'`